include/albatross/models/gp.h

```diff
@@ -58,7 +58,21 @@ struct Fit<GaussianProcessBase<CovFunc, ImplType>, FeatureType> {
     information = train_ldlt.solve(targets.mean);
   }
 
-  template <typename Archive> void serialize(Archive &archive) {
+  Fit(const std::vector<FeatureType> &features,
+      const JointDistribution &targets, const Eigen::MatrixXd &k_uu) {
+    train_features = features;
```

Comment:
want to add a brief comment explaining the rationale behind the math here?

Reply:
Added some comments, feel free to suggest improvements. :)